ZeekSpicyAnalyzerSupport.cmake

```diff
@@ -51,14 +51,14 @@ function (spicy_add_analyzer)
         list(APPEND CXX_LINK ${cxx_link})
     endforeach ()
 
-    # Note: The link below to `spicy-modules` is for backwards compatibility with older zkg packages.
+    # Create a link for backwards compatibility with older zkg packages.
+    file(CREATE_LINK "." ${SPICY_MODULE_OUTPUT_DIR_BUILD}/spicy-modules SYMBOLIC)
+
     add_custom_command(
         OUTPUT ${OUTPUT}
         DEPENDS ${SPICY_ANALYZER_SOURCES} spicyz
         COMMENT "Compiling ${SPICY_ANALYZER_NAME} analyzer"
         COMMAND mkdir -p ${SPICY_MODULE_OUTPUT_DIR_BUILD}
```

Comment:
Seems this `mkdir -p` is now superfluous given the `file(CREATE_LINK...)` probably would've bailed if it hadn't existed already?

Reply:
Hmm, and now I'm also wondering if the `spicyz` invocation just below should actually use `${SPICYZ}` instead?

I'm holding off merging this, because seems both would make sense but I'm not cmake-savy enough to just do them (and am not setup to test spicy analyzers as easy as I'd hope).

Reply:
> Seems this `mkdir -p` is now superfluous

Ah, of course, yes, that can go.